public/PlanX_logo.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><svg id="a" xmlns="http://www.w3.org/2000/svg" viewBox="73 10 30 30"><defs><style>.b,.c{fill:#000;stroke-width:0px;}</style></defs><polygon class="b" points="99.56 17.06 97.44 14.94 89 23.38 80.56 14.94 78.44 17.06 86.88 25.5 78.44 33.94 80.56 36.06 89 27.62 97.44 36.06 99.56 33.94 91.12 25.5 99.56 17.06"/></svg>
```

Comment:
Eventually probably want this to be an ODP logo rather than PlanX one but can't easily find an assets folder for ODP stuff online or in GDrive, so this works for now :+1: